packages/plugin-lighthouse/src/lib/utils.unit.test.ts

```diff
@@ -121,684 +112,250 @@ describe('validateOnlyCategories', () => {
+    })),
+    groups: groups.map(({ slug: groupSlug, title: groupTitle, refs = [] }) => ({
+      slug: groupSlug ?? 'slug',
+      title: groupTitle ?? 'title',
+      refs: refs.map(({ slug: auditRegSlug, weight } = {}) => ({
+        slug: auditRegSlug ?? 'slug',
+        weight: weight ?? 1,
+      })),
+    })),
+  });
+
+  it('should return given audits and groups if no filter is set', () => {
+    const audits = [{ slug: 'speed-index' }] as Audit[];
+    const groups = [
       {
         slug: 'performance',
-        title: 'Performance',
-        refs: [{ slug: 'speed-index', weight: 1 }],
+        refs: [{ slug: 'speed-index' }],
       },
-    ];
+    ] as Group[];
     const { audits: filteredAudits, groups: filteredGroups } =
       filterAuditsAndGroupsByOnlyOptions(audits, groups, {});
 
     expect(filteredAudits).toStrictEqual(audits);
     expect(filteredGroups).toStrictEqual(groups);
 
-    const pluginConfig: PluginConfig = {
-      slug: 'coverage',
-      title: 'Code Coverage',
-      icon: 'file',
-      description: 'Runs test coverage and created audits',
-      audits: filteredAudits,
-      groups: filteredGroups,
-      runner: {
-        command: 'node',
-        outputFile: 'out.json',
-      },
-    };
-
+    const pluginConfig = basePluginConfig(filteredAudits, filteredGroups);
     expect(() => pluginConfigSchema.parse(pluginConfig)).not.toThrow();
   });
 
-  it('should filter audits if onlyAudits is set', () => {
+  it('should filter audits if skipAudits is set', () => {
     const { audits: filteredAudits, groups: filteredGroups } =
       filterAuditsAndGroupsByOnlyOptions(
         [
-          { slug: 'speed-index', title: 'Speed Index' },
-          { slug: 'first-contentful-paint', title: 'First Contentful Paint' },
-        ],
+          { slug: 'speed-index' },
+          { slug: 'first-contentful-paint' },
+        ] as Audit[],
         [
           {
             slug: 'performance',
-            title: 'Performance',
-            refs: [{ slug: 'speed-index', weight: 1 }],
+            refs: [{ slug: 'speed-index' }, { slug: 'first-contentful-paint' }],
           },
-        ],
-        { onlyAudits: ['speed-index'] },
+        ] as Group[],
+        { skipAudits: ['speed-index'] },
       );
 
-    expect(filteredAudits).toStrictEqual<Audit[]>([
-      { slug: 'speed-index', title: 'Speed Index' },
-    ]);
-    expect(filteredGroups).toStrictEqual<Group[]>([
-      {
-        slug: 'performance',
-        title: 'Performance',
-        refs: [{ slug: 'speed-index', weight: 1 }],
-      },
-    ]);
-
-    const pluginConfig: PluginConfig = {
-      slug: 'coverage',
-      title: 'Code Coverage',
-      icon: 'file',
-      description: 'Runs test coverage and created audits',
-      audits: filteredAudits,
-      groups: filteredGroups,
-      runner: {
-        command: 'node',
-        outputFile: 'out.json',
-      },
-    };
+    expect(filteredAudits).toHaveLength(1);
```

Comment:
🧪 Test maintainability: Please rewrite this to include `expect.ObjectContaining` instead of nested property access.
This is applicable to multiple tests.

Reply:
I had that before and did it by following your last suggestion of testing each line witch a separate expect. :)